### DISCUSSION TITLE
Only allow register new users via oauth

### DISCUSSION THREAD

**Discussion by seriousm4x** [pocketbase/pocketbase#2824] [Q&A] [answered]
I would like to deny user registrations via username/password, but still allow registrations via OAuth2.

E.g. I want his request to fail:

POST http://localhost:8090/api/collections/users/records
```json
{
	"password": "12345678",
	"passwordConfirm": "12345678"
}
```

Is this possible via the api create filter rule?

**Comment by ganigeorgiev** [comment, marked as answer]
It is a little "hacky", but you can try with: `@request.data.password:isset = false` (this should result to a generic `400` error).

**Reply by seriousm4x** [reply]
Wow.. Tried a couple combinations but this didn't come up my mind. Thanks!

Does this have any other side effects?

**Reply by ganigeorgiev** [reply]
> Does this have any other side effects?

No, it shouldn't, at least I can't think of any at the moment.
